### ISSUE TITLE
Spatially Resolved Transcriptomics Deconvolutes Prognostic Histological Subgroups in Patients with Colorectal Cancer and Synchronous Liver Metastases

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#21]
## TL;DR

Strong immune responses in primary colorectal cancer correspond with better patient survival following surgery compared with tumors with predominantly stromal microenvironments. However, biomarkers to identify patients with colorectal cancer liver metastases (CRLM) with good prognosis following surgery for oligometastatic disease remain elusive. The aim of this study was to determine the practical application of a simple histological assessment of immune cell infiltration and stromal content in predicting outcome following synchronous resection of primary colorectal cancer and CRLM and to interrogate the underlying functional biology that drives disease progression.

### Paper Link

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10102851/

### Author/Institution

Colin Wood (University Department of Surgery, Glasgow Royal Infirmary)

## Overview

Strong immune responses in primary colorectal cancer correspond with better patient survival following surgery compared with tumors with predominantly stromal microenvironments. However, biomarkers to identify patients with colorectal cancer liver metastases (CRLM) with good prognosis following surgery for oligometastatic disease remain elusive. The aim of this study was to determine the practical application of a simple histological assessment of immune cell infiltration and stromal content in predicting outcome following synchronous resection of primary colorectal cancer and CRLM and to interrogate the underlying functional biology that drives disease progression.

## Methods

colorectal cancer. survival analysis

## Cite

Wood CS, Pennel KAF, Leslie H, Legrini A, Cameron AJ, Melissourgou-Syka L, Quinn JA, van Wyk HC, Hay J, Roseweir AK, Nixon C, Roxburgh CSD, McMillan DC, Biankin AV, Sansom OJ, Horgan PG, Edwards J, Steele CW, Jamieson NB. Spatially Resolved Transcriptomics Deconvolutes Prognostic Histological Subgroups in Patients with Colorectal Cancer and Synchronous Liver Metastases. Cancer Res. 2023 Apr 14;83(8):1329-1344. doi: 10.1158/0008-5472.CAN-22-2794. PMID: 37057593; PMCID: PMC10102851.
